src/renderer/modules/common/api.ts

```diff
@@ -1,4 +1,4 @@
-import { filters, getExportsForProps, waitForModule } from "..";
+import { filters, getExportsForProps, waitForModule } from "./webpack";
```

Comment:
Are you sure this is correct? I think it should be `../webpack`

Reply:
I found this to be correct.